configure.ac

```diff
@@ -717,13 +726,20 @@ AC_SUBST(OLA_CLIENT_LIBS)
 
 # Extra tools
 #####################################################
-AM_CONDITIONAL([BUILD_JA_RULE],
-               [test "x$enable_ja_rule" = xyes && test "x$have_libusb" = xyes \
-                && test "x$have_libusb_hotplug_api" = xyes])
+AS_IF([test "x$enable_ja_rule" = xyes && test "x$enable_usbdmx" != xyes],
```

Comment:
FWIW, it's --disable-usbdmx normally.

You're right about the ja_rule check, but I'd say the usbdmx one should still be = xno. I'm not sure we need to change the variable state in PLUGIN_SUPPORT unless its a case of changing it to no. It feels an auto should track through the whole way as an auto, so you can tell the intent later on.